src/pages/Store/index.jsx

```diff
@@ -34,18 +37,11 @@ export default class Store extends Component {
     if (search) {
       params.search_word = search;
     }
-    await appStore.fetchApps(params);
+    await appStore.fetchAll(params);
```

Comment:
The fetchAll defalut limit 10 datas, or add noLimit parameter，because query category mybe exceed 10 datas

Reply:
The api limit max 200 datas, show app has problem when category datas exceed 200.

Reply:
yes, first problem can be fixed by passing `{noLimit: true}`, second problem need more discuss